frontend/src/admin/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from "react";
+import DropdownMenuIcon from "../dropdown-menu-icon/dropdown-menu-icon";
+import "./dropdown-menu.scss";
+
+interface DropdownMenuProps {
+    children: React.ReactNode
+}
+
+export default function DropdownMenu( {children}: DropdownMenuProps) {
+    const [showDropdown, setShowDropdown] = useState(false);
+
+    function toggleDropdown() {
+        setShowDropdown(!showDropdown);
+    }
+
+    function hideDropdown(event: React.FocusEvent<HTMLButtonElement>): void {
+        if (event.currentTarget !== event.target) {
+            return
+        }
+        setShowDropdown(false);
+    }
+
+    return (
+        <div className={`c-dropdown-menu${showDropdown ? '--active' : ''}`}>
```

Comment:
The modifier `--active` is used incorrectly in terms of BEMIT.
The intended use of the modifier is as a separate classes: `c-dropdown-menu c-dropdown-menu--active`
That way the CSS will also be simplified because you don't need to define the samme attributes for the base class and the variation.